### ISSUE TITLE
Extension doesn't work in wsl2 environment, but work fine in original host system

### ISSUE THREAD

**Issue by uleming** [jdneo/vscode-checkstyle#338]
Apparently, it is a networking issue, at least when I click choose checkstyle version => other versions I got popup message "Network Error".
But in original host, same VScode with extension work smooth. 

So far, I wasn't' able to trace the reason. My guesses are:
> http request to get list of  checkstyle version failed (but i dig link in sources and it is work fine, using curl in wsl environment, so this is not the network error reason)

> VScode on host machine get list of checkstyles but failed to pass them into lsp server in wsl2. 

Help me out what else it could be

**Comment by jdneo** [comment]
Haven't tried it, but as a quick workaround, you can download the `checkstyle-xx.x-all.jar` to the global storage manually. In Linux, I believe it's `$HOME/.config/Code/User/globalStorage/shengchen.vscode-checkstyle`.

And then, you can just set `java.checkstyle.version` to that version number

**Comment by uleming** [comment]
Let, me try this path, I though it is same directory where default shipped with plugin 8.18 version of checktyle so it was easy to find 
```
cd $HOME/.vscode-server
find . -type file -name "checkstyle*.jar"
```
Which gave me 
```
./extensions/shengchen.vscode-checkstyle-1.4.1/server/checkstyle-10.3.4-all.jar
./extensions/shengchen.vscode-checkstyle-1.4.1/server/checkstyle-8.18-all.jar
```
Trick didn't work.

Directory tree a bit different in wsl.

Yes 
This if I copy jar file to `$HOME/.config/Code/User/globalStorage/shengchen.vscode-checkstyle` it works. Only difference path 
`$HOME/.vscode-server/data/User/globalStorage/shengchen.vscode-checkstyle`

Thanks
